Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_produce.yml

```
@@ -48,6 +48,7 @@
       - FoodBungo
       - FoodPeaPod
       - FoodPumpkin
+      - FoodCoffee
```

Note that this has been added in Frontier's code.
```suggestion
      - FoodCoffee # Frontier
```